lexer/src/main/antlr4/gov/nasa/pds/api/registry/lexer/Search.g4

```diff
@@ -10,25 +10,54 @@ comparison : FIELD operator ( NUMBER | STRINGVAL ) ;
 likeComparison : FIELD LIKE STRINGVAL ;
 operator : EQ | NE | GT | GE | LT | LE ;
 
-NOT : 'not' ;
+NOT : 'NOT' | 'not' ;
 
-EQ : 'eq' ;
-NE : 'ne' ;
-GT : 'gt' ;
```

Comment:
Hi @alexdunnjpl ,

Have you tried this option of antlr https://github.com/antlr/antlr4/blob/master/doc/options.md#caseinsensitive ?

I believe I will approve anyway, since we don't have much time for details now.

Thanks

Reply:
@tloubrieu-jpl I'd seen it, but am not familiar enough with the parsing setup to be certain that I wouldn't introduce any unintended side-effects, so proceeded with the inelegant explicit approach